packages/coordinator/tests/logging/expectations.ts

```diff
@@ -3,20 +3,30 @@ import Hapi from "@hapi/hapi"
 import {getPayloadIdentifiersFromLogs, hasAuditTag} from "./helpers"
 import {isAuditPayloadHash, isPrepareEndpointResponse} from "./types"
 import {PayloadIdentifiersValidator} from "./validation"
+import {getSHA256PrepareEnabled} from "../../src/utils/feature-flags"
+
+const SHA1_HASH_LENGTH = 40
+const SHA256_HASH_LENGTH = 64
 
 /**
  * Expects that the hash for incoming payloads is logged.
  * @param logs - the logs produced for a request to the API
  */
-const expectPayloadAuditLogs = (logs: Array<Hapi.RequestLog>): void => {
+const expectPayloadAuditLogs = (logs: Array<Hapi.RequestLog>, path: string, expectedUseSHA256?: boolean): void => {
```

Comment:
Another default value might be useful here. Then no need for the ?? operator further along. I think there was another example of this in another file that I've collapsed so now can't find...